src/50_Samples_%26_Templates/Comment_Section.html

```diff
@@ -23,9 +23,6 @@
     <link rel="canonical" href="<%host%>/samples_templates/comment_section/">
     <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
     <style amp-custom>
```

Comment:
Two formatting issues in this sample:

1. large space below the comment list
2. new comment is rendered in the same line

<img width="998" alt="screen shot 2017-02-28 at 06 22 03" src="https://cloud.githubusercontent.com/assets/380472/23393999/751e97fc-fd7e-11e6-9b2a-cacecb83a3b0.png">